server/controllers/messageController.js

```diff
@@ -0,0 +1,31 @@
+const Message = require("../models/Message");
+
+exports.create = async (req, res, next) => {
+  try {
+    const message = new Message({
```

Comment:
We should add some data validation here - is the user actually in the conversation? was any content provided?

Reply:
Great points, have added checks for both.